Tests/XMLCoderTests/NodeEncodingStrategyTests.swift

```diff
@@ -309,4 +309,27 @@ final class NodeEncodingStrategyTests: XCTestCase {
             XCTAssert(false, "failed to decode the example: \(error)")
         }
     }
+    
+
+    func testNoEmptyElements() {
+        let encoder = XMLEncoder()
+        if #available(macOS 10.13, *) {
```

Comment:
Why is this wrapped in `#available(macOS 10.13, *)`?

Reply:
Good question. I started with the previous test as a base, which had the same check around it. I've removed it.

Note that all tests in that class have the same check which dates back several years, though my Xcode is fine when I remove them. Maybe it's not be needed anymore due to this in the Package.swift file:

```
#if swift(>=5.7)
let platforms: [PackageDescription.SupportedPlatform] = [.macOS(.v10_13), .iOS(.v11), .watchOS(.v4), .tvOS(.v11)]
#elseif swift(>=5.0)
let platforms: [PackageDescription.SupportedPlatform]? = nil
#endif
```

Though not sure what'll happen in Swift versions earlier than 5.7.